website/app/styles/pages/components/application-state.scss

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+
+// COMPONENTS > APPLICATION-STATE
+
+#show-content-components-application-state {
+  .doc-application-state-flex-layout {
+    display: flex;
+    gap: 8px;
+    align-items: center;
+    justify-content: space-between;
```

Comment:
This style (justify-content: space-between;) doesn't appear necessary.

I would probably just add margin styles to space apart the text in your examples vs. making everything a flex layout. It would be simpler & require less code.